llama/unicode.cpp

```diff
@@ -28,6 +28,19 @@
 #define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING
 #endif
 
+#if defined(_WIN32)
+#define WIN32_LEAN_AND_MEAN
+#ifndef NOMINMAX
+    #define NOMINMAX
+#endif
+#include <windows.h>
+#if __GLIBCXX__
+#include <cstdio>
+#include <ext/stdio_filebuf.h>
+#include <fcntl.h>
+#endif
```

Comment:
I guess we probably don't need the headers in the `#if __GLIBCXX__` block - looks like mostly file I/O copied from the clip patch?